packages/cli/src/utils/compiler.ts

```diff
@@ -5,6 +5,7 @@ import { PluginOption, build } from 'vite';
 import { viteExternalsPlugin } from 'vite-plugin-externals';
 import { createHash } from 'crypto';
 import { getAppBridgeVersion } from './appBridgeVersion.js';
+import * as fs from 'fs';
```

Comment:
```suggestion
import { writeFileSync, readFileSync } from 'fs';
```
Use the `sync` instead of promise approach please